indico/modules/auth/blueprint.py

```diff
@@ -25,7 +28,6 @@
 
 _bp.add_url_rule('/login/', 'login', multipass.process_login, methods=('GET', 'POST'))
 _bp.add_url_rule('/login/<provider>', 'login', multipass.process_login, methods=('GET', 'POST'))
-
 _bp.add_url_rule('/logout/', 'logout', RHLogout, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/user/identities/associate', 'associate_identity', RHAssociateIdentity, methods=('GET', 'POST'))
```

Comment:
`/user/accounts/associate` (or keep `/user/identities/associate`, see the comment below - in any case it should be consistent with the rest of the account stuff).

Or should we move this outside the /user/ url space since it's kind of special (runs before the user is fully logged in)... maybe `/login/<provider>/associate` (in that case add a trailing slash to `/login/<provider>`)